client/src/components/Header/Header.js

```diff
@@ -19,11 +24,8 @@ const Header = () => {
 
   return (
     <div id="header" className="header">
+      {/* Header elements are shown in row-reverse order, right to left */}
```

Comment:
This is to take out the `WATER THE TREES` home page link on the map for mobile since the searchbar will be overlayed on top of it for now. It solves the problem of positioning the hamburger menu icon on the right without using `float: right`. Simply remove the other elements in the flexbox row.